server/mdm/apple/mobileconfig/file_vault_options.go

```diff
@@ -12,6 +12,9 @@ type FDEFileVaultOptionsPayload struct {
 	DontAllowFDEEnable    *bool  `plist:"dontAllowFDEEnable"`
 }
 
+// ContainsFDEVileVaultOptionsPayload returns true if the payload contains any FileVault options.
+// https://developer.apple.com/documentation/devicemanagement/fdefilevaultoptions
+// Fleet users are not allowed to upload such payloads because Fleet fully manages disk encryption (FileVault).
 func ContainsFDEVileVaultOptionsPayload(contents []byte) (bool, error) {
```

Comment:
Typo? (in code and comment)
```suggestion
func ContainsFDEFileVaultOptionsPayload(contents []byte) (bool, error) {
```